src/js/App/Sidenav/SectionNav.scss

```diff
@@ -40,6 +40,10 @@
             .pf-c-nav__item {
               .pf-c-nav__link{
                 padding-left: var(--pf-global--spacer--sm);
+                svg {
+                  width: 12px;
```

Comment:
Ok so rather than adding size to the SVG element itself, it's better to add font-size to the element or its parent. The SVG size is by default `1em` which is based on the font-size. Using height/width can distort icons that are not rectangles.